src/pages/checkout/shipping-method.js

```diff
@@ -103,7 +104,8 @@ export class ShippingMethodPage extends Component {
     this.setState({ loading: true })
     // update PayPal order total
     return this.props.dispatch(updatePayPalOrderTotal(this.state.payPalOrderID, this.state.purchaseUnitsReferenceID, this.props.cart)).then(() => {
-      if (!this.props.order.paymentResponseErrors.error.data) {
+      const paymentError = (this.props.order.paymentResponseErrors.error && this.props.order.paymentResponseErrors.error.data)
+      if (!paymentError) {
```

Comment:
What happens if we **do** get an error?

Reply:
Same as above, the error thats set in state is passed to the Shipping Methods component and displayed as a flash message -

![PayPal authorisation error message](https://user-images.githubusercontent.com/4486874/56891926-fc697b80-6a75-11e9-964a-bff87fa0b1c5.png)